scripts/update-vm.sh

```diff
@@ -4,17 +4,52 @@ set -e -o pipefail
 CHEFDK_VERSION=3.4.38
 DOWNLOAD_DIR=/tmp/vagrant-cache/wget
 REPO_ROOT=~/vm-setup
-CMD_LINE_FLAG=$1
+
+RUN_LIST="stations/full-node.json"
+PROVISION_ONLY=false
+VERIFY_ONLY=false
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+  key="$1"
+  case $key in
```

Comment:
Since these are all only flags (i.e. no parsing needed), I would prefer to keep the existing approach as it is much simpler and tidier -- what do you think? Is the additional complexity of positional parameter parsing really needed here?

Reply:
Initially only one parameter was given to the script CMD_LINE_FLAG, now the script also receives a second parameter RUN_LIST, which is optional and defaults to stations/full-node.json.

I always entered them in the wrong order, that's why I added this change.